app/database/database.py

```diff
@@ -1,20 +1,25 @@
 import os
 
-from sqlalchemy import create_engine
+from dotenv import load_dotenv
+from sqlalchemy import Engine
 from sqlalchemy.ext.declarative import declarative_base
-from sqlalchemy.orm import sessionmaker
+from sqlalchemy.orm import sessionmaker, Session, DeclarativeBase
 
-from dotenv import load_dotenv
+from .check_and_create_db import check_and_create_database
 
 load_dotenv()
 
-SQLALCHEMY_DATABASE_URL = os.getenv('SQL_ALCHEMY_URL')
+DB_HOST = os.getenv("POSTGRES_HOST")
+DB_PORT = os.getenv("POSTGRES_PORT")
+DB_USER = os.getenv("POSTGRES_USER")
+DB_PASS = os.getenv("POSTGRES_PASSWORD")
+DB_NAME = os.getenv("POSTGRES_NAME")
 
-engine = create_engine(SQLALCHEMY_DATABASE_URL)
+engine: Engine = check_and_create_database(DB_USER, DB_PASS, DB_HOST, DB_PORT, DB_NAME)
```

Comment:
The previous comment is referring to this. It could stay like it is in this file, but then update param names in the function definition.